api/models/spell.py

```diff
@@ -0,0 +1,229 @@
+"""
+The model for a spell.
+
+Also includes importing and some presentation logic.
+"""
+
+from django.db import models
+from django.template.defaultfilters import slugify
+from django.core.validators import MinValueValidator
+from django.core.validators import MaxValueValidator
+
+from .models import GameContent
+from ..validators import spell_school_validator
+
+
+class Spell(GameContent):
+    """The model for a spell."""
+
+    # Stored metadata about the spell
+    page = models.TextField(
+        help_text='Page number reference for the document.')
+
+    spell_level = models.IntegerField(
+        validators=[MinValueValidator(0), MaxValueValidator(9)],
+        help_text='Integer representing the level of the spell. Cantrip is 0.')
+
+    dnd_class = models.TextField(
+        help_text='List of classes (comma separated) that can learn this spell.')
```

Comment:
This seems like it should be a ManyToManyField to Class.

Reply:
Yes, I generally agree. I think there's a bit of follow up work to be done on spell that includes at a minimum bumping this out into a list, but there's implications on loading that I'm not sure I can sort out easily. Your concerns is understood, and I agree that long-term it's the best decision. Out of scope of this larger PR though.